### PR TITLE
Hlm 5288 pull impel changes

### PR DESCRIPTION
**Pull request by Ramkrishna-egov**

### CODE REVIEW

File: apps/health_campaign_field_worker_app/lib/pages/beneficiary/household_overview.dart [hunk 1]
```diff
@@ -493,13 +492,13 @@ class _HouseholdOverviewPageState
...
                                               sideEffects: sideEffectData,
                                               editMemberAction: () async {
-                                                final scannerbloc =
-                                                    context.read<ScannerBloc>();
-
-                                                scannerbloc.add(
-                                                  const ScannerEvent
-                                                      .handleScanner([], []),
-                                                );
+                                                // final scannerbloc =
```
Comment (naveenr-egov) [inline apps/health_campaign_field_worker_app/lib/pages/beneficiary/household_overview.dart:495]: Remove commented code if not required

File: apps/health_campaign_field_worker_app/lib/pages/beneficiary/household_overview.dart [hunk 2]
```diff
@@ -740,12 +739,12 @@ class _HouseholdOverviewPageState
 
                                         final projectId = context.projectId;
 
-                                        context.read<ScannerBloc>().add(
-                                              const ScannerEvent.handleScanner(
-                                                [],
-                                                [],
-                                              ),
-                                            );
+                                        // context.read<ScannerBloc>().add(
```
Comment (naveenr-egov) [inline apps/health_campaign_field_worker_app/lib/pages/beneficiary/household_overview.dart:742]: Remove commented code if not required

File: apps/health_campaign_field_worker_app/lib/widgets/sidebar/side_bar.dart [hunk 3]
```diff
@@ -90,8 +90,8 @@ class SideBar extends StatelessWidget {
             icon: Icons.home,
             onPressed: () {
               Navigator.of(context, rootNavigator: true).pop();
-              context.router.popUntilRoot();
-              context.router.push(HomeRoute());
+              context.router.replaceAll([HomeRoute()]);
+              // context.router.push(HomeRoute());
```
Comment (naveenr-egov) [inline apps/health_campaign_field_worker_app/lib/widgets/sidebar/side_bar.dart:94]: Remove commented code if not used

File: apps/health_campaign_field_worker_app/lib/pages/search_beneficiary.dart [hunk 4]
```diff
@@ -102,6 +103,20 @@ class _SearchBeneficiaryPageState
...
+                      metrics.pixels != 0) {
+                    final bloc = context.read<SearchHouseholdsBloc>();
+                    bloc.add(SearchHouseholdsEvent.searchByHouseholdHead(
+                      searchText: searchController.text,
+                      projectId: context.projectId,
+                      isProximityEnabled: isProximityEnabled,
+                      offset: offset + limit,
+                      limit: limit,
+                    ));
+                    setState(() {
```
Comment (naveenr-egov) [inline apps/health_campaign_field_worker_app/lib/pages/search_beneficiary.dart:117]: Remove setState and add it to bloc as changes made in Impel

**Review by naveenr-egov — CHANGES_REQUESTED** [review]

**Review by naveenr-egov — APPROVED** [review]
Verified Previous requested changes

**Review by naveenr-egov — DISMISSED** [review]
Reviewed changes requested.

**Review by naveen-egov — APPROVED** [review]

**Pull request merged**